src/Netatmo/Models/Client/Air/HomesCoachs/DashBoardData.cs

```diff
@@ -0,0 +1,47 @@
+using Newtonsoft.Json;
+using NodaTime;
+
+namespace Netatmo.Models.Client.Air.HomesCoachs
+{
+    public class DashBoardData
+    {
+        [JsonProperty("time_utc")]
+        Instant TimeUtc { get; set; }
+        
+        [JsonProperty("Temperature")]
+        float Temperature {get;set;}
+        
+        [JsonProperty("CO2")]
+        int CO2 {get;set;}
+        
+        [JsonProperty("Humidity")]
+        int Humidity {get;set;}
+        
+        [JsonProperty("Noise")]
+        float Noise {get;set;}
+        
+        [JsonProperty("Pressure")]
+        float Pressure {get;set;}
+        
+        [JsonProperty("AbsolutePressure")]
+        float AbsolutePressure {get;set;}
+        
+        [JsonProperty("health_idx")]
+        float HealthIdx {get;set;}
```

Comment:
The documentation say it's a enum from 0 to 4 so it's a `int` , or i miss something ?
A Test case like in `Netatmo.Tests.WeatherClient` with a sample of return can help me, because I don't have any Coach Device.

```suggestion
        // 0 = Healthy, 1 = Fine, 2 = Fair, 3 = Poor, 4 = Unhealthy
        int HealthIdx {get;set;}
```

And can you create an enum with propertylike `WifiStrength` in `Netatmo.Models.Client.Weather.StationsData.Device`